.buildkite/lint.sh

```
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [ $# -eq 0 ]; then
+  golangci-lint run
+  yamllint --format parsable .
+  cd web
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)